src/OAM_DMA_Transfer.md

```diff
@@ -58,4 +67,14 @@ run_dma_hrampart:
 ```
 
 This saves 5 bytes of HRAM, but is slightly slower in most cases due to
-the jump into the HRAM part.
+the jump to the tail in HRAM.
+
+The details:
+
+* If OAM DMA is active during OAM scan (mode 2), most PPU revisions read each object
+  as being off-screen and thus hidden on that line.
+* If OAM DMA is active during rendering (mode 3), the PPU reads whatever 16-bit word
+  the DMA unit is writing to OAM when the object is fetched.
+  This causes an incorrect tile number and attributes for objects already determined to be in range.
```

Comment:
Shouldn't this be moved above the "more compact procedure"?

Reply:
I'd prefer to move the "more compact procedure" right below the other code snippet, introduced by something to the effect "If HRAM is tight, try this more compact procedure that saves a few bytes of HRAM at the cost of a few cycles."